python/lsst/afw/display/interface.py

```diff
@@ -29,6 +29,9 @@
 import importlib
 import lsst.afw.geom  as afwGeom
 import lsst.afw.image as afwImage
+import lsst.log
+
+logger = lsst.log.Log.getLogger("afw.display.interact")
```

Comment:
While doing the logging framework transition I have the exactly same question about the convention of logger names. If the log belongs to a Task the logger name is its `_DefaultName`.  If it's not a Task, the name seems a bit arbitrary and depends on the author's intent from I've seen so far.